cmd/db-exporter/src/main.rs

```diff
@@ -828,7 +828,47 @@ pub fn export_block_range(
         ProgressStyle::default_bar()
             .template("[{elapsed_precise}] {bar:100.cyan/blue} {percent}% {msg}"),
     );
+    let mut visit = HashSet::new();
```

Comment:
Iterating the list, then collecting them to HashSet. It's more rust idiomatic.

Reply:
Got it. I will try this.

Reply:
Got it. I will try this.